src/components/ContactDetails.vue

```diff
@@ -832,13 +832,6 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-// Container height fix for dropdowns
-.app-content-details {
-	height: 100%;
-	min-height: calc(100vh - var(--header-height));
```

Comment:
Are you sure that this does not break scrolling on mobile?

Reply:
I tested it and it worked fine. The parent containers (from nc-vue) make sure of that.

Reply:
all right, cool!